geniza/corpus/tests/test_corpus_signals.py

```diff
@@ -64,3 +65,14 @@ def test_related_delete(mock_indexitems, document, join):
     assert mock_indexitems.call_count == 1
     assert document in mock_indexitems.call_args[0][0]
     assert join not in mock_indexitems.call_args[0][0]
+
+
+@pytest.mark.django_db
+def test_items_to_index(document, footnote):
+    """Test that items_to_index doesn't fail silently."""
+    # NOTE: Footnote must be included in fixtures for full prefetching to occur.
+    # NOTE: Queryset must be explicitly called to trigger prefetching error.
+    #  Other methods of accessing the queryset may not trigger anything.
```

Comment:
can you briefly clarify what/why other scenarios don't trigger the problem? 

is there anything further that should be tested here that isn't covered by other tests?